update.sh

```diff
@@ -1,6 +1,38 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
This is an unrelated change.

Reply:
Sure, but is it worth putting in a separate PR?

Reply:
Personally I will, and maybe add `[ci skip]` in commit message to skip the build.

Reply:
K, I'll drop the relevant commit